package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "prepublish": "node ./apidocs/describe-builtin-models.js",
-    "pretest": "jshint .",
-    "test": "node ./test.e2e/test-server.js node node_modules/karma/bin/karma start --single-run --browsers PhantomJS"
+    "test": "node ./test.e2e/test-server.js node node_modules/karma/bin/karma start --single-run  --browsers PhantomJS",
+    "posttest": "npm run prepublish && eslint ."
```

Comment:
Thats fine then, too bad we can't add comments to JSON. ;) Maybe add a comment in the readme regarding posttest and why we run prepublish.